README.md

```diff
@@ -43,6 +43,9 @@ through plugins.
   Kubernetes.
 - **Dynamic Load Balancing**: Load balance traffic across multiple backend
   services.
+- **Hash-based Load Balancing**: Load balance consisten hashing/sticky sessions.
+- **Circuit-Breaking**: Smarter trackling of unhealthy upstream instances.
```

Comment:
"trackling" -> "tracking"

Reply:
Also, "Circuit Breaking" isn't really a term we use - "Circuit Breaker" is more appropriate, and better SEO